### PR TITLE
Introduce configuration section

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+== Configuration
+
+MicroProfile OpenTracing project leverages MicroProfile Config specification to provide a consistent means for
+all supported configuration options. It means that vendor implementations must by also compliant with
```
Comment (objectiser) [inline spec/src/main/asciidoc/configuration.asciidoc]: "must by also" -> "must also be"

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+the Config specification.
+
+All configuration keys supported by this projects start with `mp.opentracing`.
```
Comment (objectiser) [inline spec/src/main/asciidoc/configuration.asciidoc]: "projects" -> "project"

File: spec/src/main/asciidoc/configuration.asciidoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+=== Configuration items
+
+[cols="1,4"]
+|===
+| Configuration key | Description
+
+| `mp.opentracing.server.skip-pattern` | Configuration property to specify a skip pattern to avoid tracing of selected REST endpoints. See <<server-skip-pattern>>.
```
Comment (objectiser) [inline spec/src/main/asciidoc/configuration.asciidoc]: May be drop "Configuration property to" from each entry?

Comment (pavolloffay) [reply spec/src/main/asciidoc/configuration.asciidoc]: changed to `Specifies a skip pattern `...

**Comment by pavolloffay** [comment]
thanks @objectiser

**Pull request merged**